test/controllers/api2_controller_test.rb

```diff
@@ -354,6 +354,8 @@ def test_post_user
     assert_equal(CGI.escapeHTML("<p>New API2 Key</p>"), notes.to_s)
   end
 
+  # FIXME: does this test also need to be altered to test for QueuedEmail?
+  # Why does it pass if not?
   def test_post_api_key
```

Comment:
This test checks mail deliveries, but `api_key_api.rb` has been changed to use `QueuedEmail` now. 

Why does this test still pass? Is it because `QueuedEmail.queue` is not set to `true` and it sends right away?
Should I leave it as is (you can test more things in the content of an actual `ActionMailer::Base.deliveries`!) or should I also test that it's going into the queue?

Reply:
@pellaea This is my last remaining question on the PR. 

`api2_controller_test#test_post_api_key` is checking for emails in `ActionMailer::Base.deliveries`, and it passes. But the emails are being sent via QueuedEmail now. What's going on, do you think?

Reply:
Ah, nevermind! It does indeed stop working if i turn on the queue.
Adding a line in the test to keep the queue off for this test, in case it ever gets left on and there are mysterious errors.